plugins/browser-plugin-error-tracking/src/util.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Truncate stack trace if it's size is greater than 8kb
+ *
+ * @param stackTrace - The stack trace to truncate
+ * @returns The trucated stack trace
+ */
+export function truncateStackTrace(stackTrace?: string): string | undefined {
+  const byteSize = (str: string) => new Blob([str]).size;
```

Comment:
I think this is complicating it too much 😅 The `stackTrace` property in the schema is a string with max length 8192 characters. Converting it to bytes and setting the limit in bytes is a bit unintuitive.

You could just take the prefix of the string – take the first 8192 characters if the string has more than that. See how we do it in [the mobile trackers](https://github.com/snowplow/snowplow-ios-tracker/blob/master/Sources/Snowplow/Events/TrackerError.swift#L67). Btw, there are also limits on the message and exception name that would be good to check as well.

Reply:
Ah ok, I thought it was actually limited in bytes, so this was taking into account multibyte characters!